content/docs/integrations/istio/sidecar/ingress.md

```diff
@@ -20,70 +20,66 @@ The open source project Istio is the leading service mesh implementation that of
+1. Get the Helm values for your current Helm installation. 
    ```sh
-   helm get values kgateway -n kgateway-system -o yaml > kgateway.yaml
-   open kgateway.yaml
+   helm get values {{< reuse "/docs/snippets/helm-kgateway.md" >}} -n {{< reuse "docs/snippets/namespace.md" >}} -o yaml > {{< reuse "/docs/snippets/helm-kgateway.md" >}}.yaml
+   open {{< reuse "/docs/snippets/helm-kgateway.md" >}}.yaml
    ```
    
-2. Add the following values to the Helm values file to enable the Istio integration in kgateway.
-   ```yaml
+2. Add the following values to the Helm values file to enable the Istio integration in {{< reuse "/docs/snippets/kgateway.md" >}}.
+   ```yaml   
    controller:
      extraEnv:
        KGW_ENABLE_ISTIO_AUTO_MTLS: true
    ```
    
-3. Upgrade your kgateway installation. This upgrade automatically triggers a restart of any existing gateway proxies to inject `istio-proxy` and `sds` containers.
+3. Upgrade your Helm installation. This upgrade automatically triggers a restart of any existing gateway proxies to inject `istio-proxy` and `sds` containers.
    
    ```sh
-   helm upgrade -i --namespace kgateway-system --version v2.0.0-main kgateway oci://cr.kgateway.dev/kgateway-dev/charts/kgateway -f kgateway.yaml
-   ```
-
-4. Check that the gateway proxy automatically restarts with two additional containers, `istio-proxy` and `sds`. 
-   ```sh
-   kubectl get pods -n kgateway-system
+   helm upgrade -i --namespace {{< reuse "docs/snippets/namespace.md" >}} --version {{< reuse "/docs/versions/helm-version-flag.md" >}} {{< reuse "/docs/snippets/helm-kgateway.md" >}} oci://{{< reuse "/docs/snippets/helm-path.md" >}}/charts/{{< reuse "/docs/snippets/helm-kgateway.md" >}} -f {{< reuse "/docs/snippets/helm-kgateway.md" >}}.yaml
    ```
    
    {{< callout type="warning" >}}
    The `istio-proxy` container in the gateway proxy looks for a service that is named `istiod` in the `istio-system` namespace to obtain a valid certificate. Depending on how you installed Istio, you might have a revisioned istiod deployment, such as `istiod-main`, or custom names for the Istio meta cluster ID and meta mesh ID. If this is the case, the `istio-proxy` container cannot deploy successfully. Continue with [Revisioned istiod and custom Istio meta mesh settings](#custom-istio-settings) to configure the `istio-proxy` container to use your custom values. 
    {{< /callout >}}
-   
 
-### Revisioned istiod and custom Istio meta mesh settings {#custom-istio-settings}
+### Update the Istio proxy settings {#custom-istio-settings}
 
-If you installed Istio with a revision, or you set up a custom Istio meta cluster ID and meta mesh, you must create a GatewayParameters resource to change the default Istio settings on your gateway proxy. 
+Create a GatewayParameters resource to configure the Istio SDS container to pull the image from the kgateway repository. The steps vary depending on if you used revisioned Istio with custom cluster and mesh IDs or not.
```

Comment:
so if i used revisions OR custom cluster mesh IDs, I use the first tab? and no revisions AND no custom IDs is second tab? unsure which requirements are needed for each

Reply:
updated this